lib/core/utils/flattened-tree.js

```diff
@@ -46,6 +46,18 @@ function virtualDOMfromNode(node, shadowId) {
 				);
 			}
 			return vNodeCache._tabbableElements;
+		},
+
+		// abstract Node and Element APIs so we can run axe in DOM-less
+		// environments. these are static properties in the assumption
+		// that axe does not change any of them while it runs.
+		elementNodeType: node.nodeType,
```

Comment:
VirtualNode objects always reference Element instances. This will always be `nodeType === 1`. I don't think we need this.

Reply:
This turns out not to be true. Text elements make it into the virtual tree and excluding them as such result in tons of errors in the tests